lib/Client.js

```diff
@@ -1423,6 +1423,30 @@ class Client extends EventEmitter {
         }).then((guild) => new Guild(guild, this));
     }
 
+    /**
+    * Edit guild channels' positions. Note that channel position numbers are lowest on top and highest at the bottom.
```

Comment:
```suggestion
    * Edit guild channel positions. Note that channel position numbers are lowest on top and highest at the bottom.
```
grammar police alert

Reply:
also channel permissions are a bit niche, and requires in depth documentation for it to be explained properly. your wording isn't wrong, but it also isn't right. maybe we can sort this out sometime in the future